src/modules/cpu/cpu.c

```diff
@@ -68,7 +71,12 @@ void ffPrintCPU(FFCPUOptions* options)
             if(coreTypes.length > 0)
                 ffStrbufAppendF(&str, " (%s)", coreTypes.chars);
             else if(cpu.coresOnline > 1)
-                ffStrbufAppendF(&str, " (%u)", cpu.coresOnline);
+            {
+                if(cpu.cpuCount > 1)
+                    ffStrbufAppendF(&str, " (%u)", cpu.coresOnline / 2);
```

Comment:
`cpu.coresOnline / cpu.cpuCount`?

What if you disable some cores in BIOS?

Reply:
Good question. Things like these is why I was not 100% sure if it's better to do it like this or just leave as it is now (total sum between the processor packages). The alternative could be doing a line for each CPU (like it would be done for laptops with iGPU and dGPU) and then listing each CPU's coresOnline?

Reply:
I prefer leaving as it is now

Reply:
Another question: is an [AMD CCD](https://www.reddit.com/r/Amd/comments/io3twa/what_is_ccdcore_complex_die/) considered a package?

Reply:
> Another question: is an [AMD CCD](https://www.reddit.com/r/Amd/comments/io3twa/what_is_ccdcore_complex_die/) considered a package?

No, on Linux, the entire CPU is considered a "Physical CPU", rather than a CCD being considered a "Physical CPU". I confirm that all cores of AMD EPYC 9754 CPU (128 cores) are marked with id 0. But I'm not sure about the implementation in windows because I don't have a corresponding windows device.